### PR TITLE
Add GitHub Actions so we can move from Travis

### PR DESCRIPTION
**Pull request by amakarudze**
Travis is causing several build errors for issues that should not fail. This will enable us to move from Travis CI to GitHub Actions.

### CODE REVIEW

File: .github/workflows/django.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          POSTGRES_DB: djangogirls
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```
Comment (carltongibson) [inline .github/workflows/django.yml:35]: Do we need all these versions? 🤔

**Review by carltongibson — APPROVED** [review]
Looks good. I think we should go with it. Thanks @amakarudze!

**Comment by carltongibson** [comment]
@amakarudze Did you have any outstanding questions of this? (I think we should give it a go.)

File: .github/workflows/django.yml (hunk 1 above)
Comment (amakarudze) [reply .github/workflows/django.yml:35]: We are using 3.4 in production, was using 3.5 in Travis and we might want to move to Django 2.2 or higher and a higher version of Python. How do we handle all those variations?

Comment (carltongibson) [reply .github/workflows/django.yml:35]: I'd suggest one at a time... 

It's ok to run all these. Let's see if it's compatible, then we can drop the older ones as we update.

**Pull request merged**